tests/etl/test_job_loader.py

```diff
@@ -104,10 +104,6 @@ def test_ingest_pending_pulse_job(pulse_jobs, result_set_stored,
     assert job.taskcluster_metadata
     assert job.taskcluster_metadata.task_id == 'IYyscnNMTLuxzna7PNqUJQ'
 
-    # should not have processed any log or details for pending jobs
-    assert JobLog.objects.count() == 0
```

Comment:
I think these asserts should be kept, but updated to the correct count and the comment changed to say something like "log URLs and job details should be processed even if the job is pending".